packages/bitski/src/-private/network.ts

```diff
@@ -18,6 +18,16 @@ export const Sepolia: Network = {
   rpcUrl: 'https://api.bitski.com/v1/web3/sepolia',
 };
 
+export const Base: Network = {
+  chainId: 8453,
+  rpcUrl: 'https://api.bitski.com/v1/web3/base',
+};
+
+export const BaseGoerli: Network = {
+  chainId: 84531,
+  rpcUrl: 'https://api.bitski.com/v1/web3/basegoerli',
```

Comment:
this should be `basegor` to match the short name here https://github.com/BitskiCo/bitski-rs/blob/main/bitski-chain-models/src/chains.json#L12611, or just the chain id